src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  useEffect,
+} from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { CSSTransition } from 'react-transition-group';
+import { ErrorType } from '../../types/ErrorType';
+import { AppDispatch, RootState } from '../../redux/store';
+import { clearErrorType, hideError } from '../../redux/todoSlice';
+import { showErrorWithTimeout } from '../../redux/showErrorThunk';
+
+export const ErrorNotification: React.FC = () => {
+  const dispatch = useDispatch<AppDispatch>();
+  const errorType = useSelector((state: RootState) => state.todos.errorType);
+  const isErrorVisible = useSelector(
+    (state: RootState) => state.todos.isErrorVisible,
+  );
+
+  useEffect(() => {
+    if (errorType) {
+      dispatch(showErrorWithTimeout());
+    }
+  }, [errorType, dispatch]);
+
+  const handleHideError = () => {
+    dispatch(hideError());
+    dispatch(clearErrorType());
+  };
+
+  const getErrorMessage = (error: ErrorType | null): string | null => {
+    return error;
+  };
```

Comment:
your helper does nothing here. you can just get error message like this: ErrorType.loadError or something.